sql/src/main/java/io/crate/protocols/postgres/types/PGTypes.java

```diff
@@ -98,7 +98,9 @@
         PG_TYPES_TO_CRATE_TYPE.put(0, DataTypes.UNDEFINED);
         PG_TYPES_TO_CRATE_TYPE.put(TEXT_OID, DataTypes.STRING);
         PG_TYPES_TO_CRATE_TYPE.put(TEXT_ARRAY_OID, new ArrayType(DataTypes.STRING));
+        // PG_TYPES_TO_CRATE_TYPE.put(VarCharType.NameType.OID, DataTypes.STRING);
```

Comment:
Can we remove this and/or add a coment explaining **why** we don't add it to the map?

Reply:
done